frontend/src/components/eMIB/HowTo.jsx

```diff
@@ -8,12 +8,18 @@ import Evaluation from "./Evaluation";
 import ProgressPane from "../commons/ProgressPane";
 import SideNavigation from "../commons/SideNavigation";
 
-const SPECS = [
-  { id: 0, text: LOCALIZE.emibTest.howToPage.overview.title, body: <Overview /> },
-  { id: 1, text: LOCALIZE.emibTest.howToPage.tipsOnTest.title, body: <TipsOnTest /> },
-  { id: 2, text: LOCALIZE.emibTest.howToPage.testInstructions.title, body: <TestInstructions /> },
-  { id: 3, text: LOCALIZE.emibTest.howToPage.evaluation.title, body: <Evaluation /> }
-];
+export const specsDefinition = () => {
```

Comment:
can we name this something that sounds like a function instead of a constant? ex - `getHowToPages` or `getHowToDefinitions`

It'd also be useful to have a comment explaining what this array is for. ex - '// Returns array where each item indicates specifications for a pre-test how to page including the title and body.

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/92/commits/3544261ab9dbd9c7a4a3137c7de2753650a5caaf